lib/bulkPrefixer.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
I'm not quite clear on the intent this function.  Could you add a comment as to what this does and why we need it?